public/index.html

```diff
@@ -19,7 +19,17 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>fortissimo Breakfast Software</title>
```

Comment:
Since this is fortissimo's landing page, let's just do `<title>fortissimo</title>`.